kernel.rb

```diff
@@ -26,4 +26,28 @@ module Kernel
   def clone(freeze: nil)
     __builtin_rb_obj_clone2(freeze)
   end
+
+  private
```

Comment:
`Kernel.Float` is a module function, which defines also a public class method not only a private instance method.